packages/ai/src/lib/useTldrawAi.ts

```diff
@@ -1,8 +1,22 @@
 import { useCallback, useMemo, useRef } from 'react'
-import { Editor, uniqueId, useMaybeEditor } from 'tldraw'
+import {
+	Editor,
+	exhaustiveSwitchError,
+	TLShapeId,
+	TLShapePartial,
+	uniqueId,
+	useMaybeEditor,
+} from 'tldraw'
 import { TldrawAiModule, TldrawAiModuleOptions } from './TldrawAiModule'
 import { TLAiChange, TLAiPrompt, TLAiSerializedPrompt } from './types'
 
+/** @public */
+export interface TldrawAi {
+	prompt(message: TldrawAiPromptOptions): { promise: Promise<void>; cancel(): void }
+	repeat(): { promise: Promise<void>; cancel: (() => void) | null }
+	cancel(): void
+}
+
```

Comment:
This might be a confusing type / concept to include, seeing as we already have things like TldrawAiModule, etc...

Reply:
It's fine I documented it.